tests/test_runner/test_parallel.py

```diff
@@ -10,6 +10,17 @@
     tblib = None
 
 
+class ExceptionThatFailsUnpickling(Exception):
+    """
+    This class fails unpickling with an error about incorrect arguments passed
+    to __init__().
+    """
+    def __init__(self, arg1, arg2):
+        self.arg1 = arg1
+        self.arg2 = arg2
+        super(ExceptionThatFailsUnpickling, self).__init__('Message with %s and %s')
```

Comment:
Does the format string `'Message with %s and %s'` serve a purpose here? If so, a comment might be good. It doesn't seem like it is necessary.